pkg/types/ccip/models.go

```diff
@@ -0,0 +1,25 @@
+package ccip
+
+import (
+	"encoding/hex"
+	"strings"
+)
+
+type Address string
```

Comment:
Can the libocr `Account` type be used instead? So that we don't create two different standards?

Reply:
This is all part of one unified relayer API, so there is no sense having two different types for the same thing. An alias can work to bridge the existing code, but we should keep that as an implementation detail in core. There is no need for this common repo to inherit and spread that complexity from the service interface layer.